core/src/main/resources/config.yml

```diff
@@ -115,6 +115,12 @@ economy:
   # Used to replace %currency% for the language files and the 2nd hologram line.
   server-currency: $
   use-xp: false
+  #taxes in %. Leave 0 if you don't want taxes. Taxes when a player buy from a chest.
+  taxes-shop-player: 15
+  #taxes when a player sells to a chest.
+  taxes-player-shop: 10
+  #This is the name of the player which takes all taxes. Leave [] if you want to burn taxes money.
```

Comment:
While this comment explains what the leader is for, sadly our `config.yml` upgrade system does not account for comments, so a config that was upgraded from an older version would not get this information.

Also, I'd leave it at `[]` by default, due to previously mentioned issues (in above comments).